### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Weyinmi24**
third assignment

### CODE REVIEW

File: Workspace/index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
   </head>
   <body>
     Hello world, My name is weyinmi Eyoyibo and i am an aspiring front-end
-    developer
+    developer My goal is to become a seasoned front-end web developer.
```
Comment (george-misan) [inline Workspace/index.html:10]: add a full spot (.) after the 'developer' before the 'My'

**Review by george-misan — CHANGES_REQUESTED** [review]

File: Workspace/index.html (hunk 1 above)
Comment (Weyinmi24) [reply Workspace/index.html:10]: Done

Comment (george-misan) [reply Workspace/index.html:10]: have you pushed the change? because I cant see it yet.

Comment (Weyinmi24) [reply Workspace/index.html:10]: i have now

**Pull request merged**